docs/001_develop/03_client-capabilities/011_criteria/04_form-criteria-builder.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: 'Form Criteria builder'
+sidebar_label: 'Form Criteria builder'
+id: form-criteria-builder
+keywords: [foundation, ui, comms, criteria, filtering]
+tags:
+  - connected
+  - ui
+  - comms
+  - criteria
+  - filtering
+---
+
+The `Form

```

Comment:
Have you missed checking something in or is this page unneccary?

Reply:
Not really clear on what this class does (or if it is even that useful, leaving the page off for now)